e2e-tests/tests/dbbackup_pg/03-assert.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
```

Comment:
I recommend adding another step which deletes `DatabaseClusterBackup` and checks if the `PerconaPGBackup` has been delete properly as well.

Reply:
The deletion tests are added